docs/api/wanchain.md

```diff
@@ -21,9 +21,9 @@ A `class` providing communication with the Wanchain blockchain using the HTTP/HT
 | options.cache | A `string` representing request cache type. It defaults to `no-cache`. Please see more details [here](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch).
 | options.credentials | A `string` representing request credentials. It defaults to `omit`. Please see more details [here](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch).
 | options.gasPriceMultiplier | A `number` represents a multiplier of the current gas price when performing a mutation. It defaults to `1.1`.
-| options.gatewayConfig.assetLedgerDeployOrderId | A `string` representing an Ethereum address of the [asset ledger deploy gateway](/#public-addresses).
-| options.gatewayConfig.actionsOrderId | A `string` representing an Ethereum address of the [actions order gateway](/#public-addresses).
-| options.gatewayConfig.valueLedgerDeployOrderId | A `string` representing an Ethereum address of the [value ledger deploy gateway](/#public-addresses).
+| options.gatewayConfig.assetLedgerDeployOrderId | A `string` representing an Wanchain address of the [asset ledger deploy gateway](/#public-addresses).
+| options.gatewayConfig.actionsOrderId | A `string` representing an Wanchain address of the [actions order gateway](/#public-addresses).
+| options.gatewayConfig.valueLedgerDeployOrderId | A `string` representing an Wanchain address of the [value ledger deploy gateway](/#public-addresses).
```

Comment:
| options.gatewayConfig.valueLedgerDeployOrderId | A `string` representing a Wanchain address of the [value ledger deploy gateway](/#public-addresses).